docs/reference/configuration/external dependencies/irsa-configuration.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Configure UDS Core to Use IAM Roles for Service Accounts (IRSA) 
+---
+
+Several applications within UDS Core can be configured to utilize resources that are external to your Kubernetes cluster, such as object storage and databases. If you are running in AWS, you can leverage [IRSA](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to provide applications within UDS Core with a secure means of accessing external resources.
+
+The following applications in UDS Core that support IRSA are:
+- [Loki](#loki)
+- [Velero](#velero)
+
+:::note
+There may be additional applications in UDS Core that are not mentioned in this list that support IRSA. Only the applications listed above have been validated to work within UDS Core.
```

Comment:
```suggestion
There may be additional applications in UDS Core that are not mentioned in this list that support IRSA. Only the applications listed above have been validated for IRSA support within UDS Core.
```
Maybe just my reading but `to work` feels more generic than the intent of "only these have been validated with IRSA".